src/WebJobs.Extensions.DurableTask/DurableOrchestrationContextBase.cs

```diff
@@ -410,6 +410,22 @@ public virtual Task CreateTimer(DateTime fireAt, CancellationToken cancelToken)
         /// </exception>
         public virtual Task WaitForExternalEvent(string name, TimeSpan timeout) => this.WaitForExternalEvent<object>(name, timeout);
 
+        /// <summary>
+        /// Waits asynchronously for an event to be raised with name <paramref name="name"/>.
+        /// </summary>
+        /// <remarks>
+        /// External clients can raise events to a waiting orchestration instance using
+        /// <see cref="DurableOrchestrationClient.RaiseEventAsync(string, string, object)"/> with the object parameter set to <c>null</c>.
```

Comment:
Not sure if this feedback is necessary, as I think it is fairly clear from the context of the signature here, but it might be a good idea to call out explicitly how the timeout and cancellation token interact. If you do decide to add that description to this comment, be sure to change it in the parameterized version of the method below.

Reply:
Ah yes, I should have updated the comments here, instead of just adding `<param>`s.  Will update. :)